### ISSUE TITLE
Cracked count on website

### ISSUE THREAD

**Issue by xTYx728** [RealEnder/dwpa#49]
Falls under enhancement not issue/bug.

Having a highscore tab on the website to show what user is cracking the most.
Would give incentive to people to run the cracker more or run custom dict's to see their name rise to the top

Just a suggestion, please close out if the team is fully against this. Thank you :)

**Comment by mrseeker** [comment]
I would add a "priority" list to this: The more work you submit, the more credits you get for a "priority" pass, where every credit will push one uncracked hash to the front of the queue. That is basically a win-win: The more submissions that yield a password, the more likely you will get rewarded with a fastpass.

**Comment by Banthex** [comment]
Currenty im working on this. Im no software developer so this takes its time. The help_crack.py is already working with a database.  I try to create a leaderboard with django cms and connect it to my database.

**Comment by nkvname** [comment]
Any hope this feature will be implemented?

**Comment by S0ly** [comment]
yeah would be nice to have an API key for your worker so you know your own stats etc

**Comment by RealEnder** [comment]
Have an idea how to approach this, but first think to add dynamic dicts, based on PROBEREQUEST frames.
